R/assets.R

```diff
@@ -42,12 +53,15 @@ earth_asset <- function(key, lon, lat, start_date, end_date = Sys.Date()){
                   "lon=", lon, "&",
                   "lat=", lat, "&",
                   "begin=", start_date, "&",
-                  if(difdate){paste0("end=",end_date,
-                                     "&api_key=",key)
-                    }else{paste0("&api_key=",key)})
-  s <- jsonlite::fromJSON(query)
+                  if (difdate) {
+                    paste0("end=", end_date, "&api_key=", key)
+                  } else {
+                    paste0("&api_key=",key)
+                  })
+
+  s <- fromJSON(query)
```

Comment:
from technical point of view, calling `fromJSON` instead of `jsonlite::fromJSON` saves some milliseconds.